src/Components/HospitalList/HospitalList.css

```diff
@@ -0,0 +1,13 @@
+.row {
+    padding: 20px 50px;
+    display: flex;
+    justify-content: space-between;
+}
+
+.row:nth-child(odd) {
+    background-color: #f7f7f7;
```

Comment:
better to use variables instead of inline colors.
so can be reused

Reply:
Maybe you can provide some example on how we can use variables in css? Didn't had experience with it before